package.json

```diff
@@ -4,7 +4,7 @@
   "description": "S3 connector",
   "main": "index.js",
   "engines": {
-    "node": "6.9.5"
+    "node": ">=6.9.5"
```

Comment:
Is it normal?

Reply:
It's not required for the release branch update but seemed like an efficient way to fix this. Because metadata is using yarn, to update our node version at all we need to have this in any project that is a dependency of metadata so we should be consistent across the projects.